libs/wingsdk/src/util/util.ts

```diff
@@ -259,9 +270,19 @@ export class Util {
   /**
    * Suspends execution for a given duration.
    * @param delay The time to suspend execution.
-   * @inflight
+   * @inflightImpl __sleep__inflight
    */
-  public static async sleep(delay: Duration): Promise<void> {
+  public static sleep(delay: Duration): void {
+    Atomics.wait(
+      new Int32Array(new SharedArrayBuffer(4)),
+      0,
+      0,
+      delay.seconds * 1000
+    );
+  }
```

Comment:
With https://github.com/winglang/wing/pull/6157, this approach will no longer work.
I think the only "reasonable" way left would be doing it via a shell.